sirepo/job_api.py

```diff
@@ -100,7 +108,14 @@ async def api_downloadDataFile(
                     req_data=req.req_data,
                     suffix=s,
                 )
-                assert not r.state == "error", f"error state in request=={r}"
+                if r.state == sirepo.job.CANCELED:
+                    # POSIT: Users can't cancel donwloadDataFile. So canceled means there was a
+                    # timeout (max_run_secs exceeded).
+                    _raise_content_too_large(req)
+                if r.state == sirepo.job.ERROR:
+                    if r.get("errorCode") == sirepo.job.ERROR_CODE_RESPONSE_TOO_LARGE:
+                        _raise_content_too_large(req)
```

Comment:
I think `raise _content_too_large(req)` is probably better, because the stack trace will be more useful. It was a mistake to have the `raise_` functions in sirepo.util.